### PR TITLE
Webdataset demo for vision module and pyarrow for bert

### PR DESCRIPTION
**Pull request by sampathkumaran90**
Have added the webdataset feature for the vision module and pyarrow - parquet format for bert module

### CODE REVIEW

File: data_prep_step/bert/data_prep_bert_pytorch.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    logging.getLogger().setLevel(logging.INFO)
+
+    PrintOptions(options)
+
+    run_pipeline(
+        options
+    )
+
+
+# if __name__ == "__main__":
```
Comment (chauhang) [inline data_prep_step/bert/data_prep_bert_pytorch.py]: @sampathkumaran90  Don't we need some version of main to launch the actual component code?

File: data_prep_step/resnet/setup.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+  def run(self):
+    for command in CUSTOM_COMMANDS:
+      self.RunCustomCommand(command)
+
+#    'apache-beam[gcp]==2.27.*',
+
+
+REQUIRED_PACKAGES = [
+    'tensorflow==2.3.*',
```
Comment (chauhang) [inline data_prep_step/resnet/setup.py]: @sampath-ideas2it Is tensorflow needed still?

File: training_step/resnet/model.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import pytorch_lightning as pl 
+import torch
+from torch import nn
+import torch.nn.functional as F
+from torchvision import transforms, models 
+
+class CIFARCNN(pl.LightningModule):
```
Comment (chauhang) [inline training_step/resnet/model.py]: Is this file getting used now?

File: training_step/bert/component_template.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
+name: train_pytorch_cnn
```
Comment (chauhang) [inline training_step/bert/component_template.yaml]: Please change the name and description to reflect the actual bert example

File: training_step/bert/component.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
+name: train_pytorch_cnn
+description: Train PyTorch ImageNet CNN
```
Comment (chauhang) [inline training_step/bert/component.yaml]: Please change the name and description to match the bert example

File: training_step/resnet/train.py [hunk 6]
```diff
@@ -0,0 +1,330 @@
...
+    )
+
+    trainer.fit(model, dm)
+    trainer.test()
+
+    return checkpoint_callback.best_model_path
+
+
+# if __name__ == "__main__":
+#     train_model(
```
Comment (chauhang) [inline training_step/resnet/train.py]: How is the train step getting launched? Isn't the main invocation needed?

File: data_prep_step/bert/data_prep_bert_pytorch.py (hunk 1 above)
Comment (sampathkumaran90) [reply data_prep_step/bert/data_prep_bert_pytorch.py]: Removed the main commented code. It was added for debugging purposes

File: data_prep_step/resnet/setup.py (hunk 2 above)
Comment (sampathkumaran90) [reply data_prep_step/resnet/setup.py]: No we don't need the tensorflow modules

File: training_step/resnet/model.py (hunk 3 above)
Comment (sampathkumaran90) [reply training_step/resnet/model.py]: No, it has been removed now

File: training_step/bert/component_template.yaml (hunk 4 above)
Comment (sampathkumaran90) [reply training_step/bert/component_template.yaml]: The name changes have been done

File: training_step/bert/component.yaml (hunk 5 above)
Comment (sampathkumaran90) [reply training_step/bert/component.yaml]: Name changes have been done

File: training_step/resnet/train.py (hunk 6 above)
Comment (sampathkumaran90) [reply training_step/resnet/train.py]: Training step is invoked from the pipeline. Removed the debugging code

**Pull request merged**